packages/insaner/src/types.ts

```diff
@@ -15,6 +16,7 @@ export type InsanerCollectorOptions = {
 
 export type NormalizedOptions = {
   level: LogLevel;
+  errorLevel: LogLevel;
```

Comment:
+ Apollo to dela taky stejne nedobre

Reply:
jak to myslis, jakoze to chces jinak nez takhle?

Reply:
ApolloCollector dela uplne stejne nedobre jako expressCollector to, ze to haze do Error a nemuzes si to nastavit

Reply:
coz se timhle resi, correct?

Reply:
neresi.
Upravujes tady express a chces upravit i apollo...

Reply:
ay! mea culpa, myslel jsem ze mluvis o apollo _serveru_, nejak jsem nepochopil ze mluvis o nasem collectoru.. fixed now, thx